cypress/e2e/API/customers.cy.js

```diff
@@ -0,0 +1,87 @@
+describe('API endpoint customers', () => {
+
+            it('Verificar se e possivel fazer a request em customers', () => {
+              cy.request(`${Cypress.env('baseUrlApi')}/customers`).then((response) => {
+                expect(response.status).to.eq(200)
+              })
+            })
+          
+            it('Verificar se e possivel paginar a lista de clientes corretamente', () => {
+
+
+              cy.request('GET', `${Cypress.env('baseUrlApi')}/customers?page=2&limit=5`)
+              .then(({ body }) => {
+                expect(body.customers).to.have.length(5)
+                expect(body.pageInfo.currentPage).to.eq('2')
+              })
+
+
+
+            })
+
+            it('Verificar se e possivel Filtrar clientes por um tipo de tamanho corretamente', () => {
+              cy.apiFilterBySize({ size: 'Small' })
+            })
+
+            it('Verificar se e possivel listar as propriedades dos customers', () => {
+              cy.request(`${Cypress.env('baseUrlApi')}/customers`).then((response) => {
```

Comment:
Que tal usar desestruturação de objetos?

Veja:

```suggestion
              cy.request(`${Cypress.env('baseUrlApi')}/customers`).then(({ body }) => {
```